velox/duckdb/conversion/DuckConversion.cpp

```diff
@@ -48,6 +48,17 @@ variant decimalVariant(const Value& val) {
     case ::duckdb::PhysicalType::INT64: {
       return variant(val.GetValueUnsafe<int64_t>());
     }
+    case ::duckdb::PhysicalType::VARCHAR: {
```

Comment:
Would you add a test for this logic to velox/duckdb/conversion/tests/DuckConversionTest.cpp ?

Reply:
Updated. Thanks.